pkg/scalers/kubernetes_workload_scaler.go

```diff
@@ -55,8 +54,13 @@ func parseWorkloadMetadata(config *ScalerConfig) (*kubernetesWorkloadMetadata, e
 	if err != nil || meta.podSelector.String() == "" {
 		return nil, fmt.Errorf("invalid pod selector")
 	}
-	meta.value, err = strconv.ParseInt(config.TriggerMetadata[valueKey], 10, 64)
-	if err != nil || meta.value == 0 {
+  valueNum, err := kedautil.ParseNumeric(config.TriggerMetadata[valueKey], 64)
+	if err != nil {
+		return nil, err
+	}
+  var ok bool
+  meta.value, ok = valueNum.(int64)
```

Comment:
just adjusting spacing
```suggestion
        valueNum, err := kedautil.ParseNumeric(config.TriggerMetadata[valueKey], 64)
	if err != nil {
		return nil, err
	}
        var ok bool
        meta.value, ok = valueNum.(int64)
```

Reply:
I'm not sure I'm parsing your suggestion.  Won't that indentation cause linting errors?  Weirdly, this isn't how it looks in my vim, there are proper tabs as far as I see.

Reply:
when I made that comment, there was no indentation whatsoever in my diff for this section. now, it's indented "too much". no idea what happened, but anyway it's fine now!